aptos-move/framework/aptos-framework/sources/primary_fungible_store.move

```diff
@@ -138,6 +138,13 @@ module aptos_framework::primary_fungible_store {
         fungible_asset::deposit(store, fa);
     }
 
+    /// Deposit fungible asset `fa` to the given account's primary store.
+    public(friend) fun force_deposit(owner: address, fa: FungibleAsset) acquires DeriveRefPod {
+        let metadata = fungible_asset::asset_metadata(&fa);
+        let store = ensure_primary_store_exists(owner, metadata);
```

Comment:
why is this called force when we check if it exists?

also this ensure command feels better to be called borrow_store or something

Reply:
force is in regard to frozen store but not the existence of a store.
The `ensure` command has been publicized... hard to change.